README.md

```diff
@@ -1,2 +1,13 @@
-# template-repository-conference
-template repository for conference proceedings
+# Monitoring CO2 sequestration with electromagnetics in the presence of steel-cased wells
+
+_Lindsey J. Heagy and Douglas W. Oldenburg_
+
+![thumbnail](./abstract/thumbnail.png)
```

Comment:
It looks like this isn't showing up. If we update it to `./presentation/AGUcasing2022.png` that should do the trick!